moped-editor/src/utils/activityLogFormatters/mopedPhasesActivity.js

```diff
@@ -53,12 +53,9 @@ export const formatPhasesActivity = (change, phaseList, subphaseList) => {
   // loop through fields to check for differences, push label onto changes Array
   Object.keys(newRecord).forEach((field) => {
     if (newRecord[field] !== oldRecord[field]) {
-      // phase end and start confirmed are in the moped_proj_phases table but not in the UI
-      // ignore changes to that field until its visually represented to users
-      if (
-        field !== "is_phase_end_confirmed" &&
-        field !== "is_phase_start_confirmed"
-      ) {
+      // filter out fields that are not listed in the activity log table maps to prevent
+      // automated field updates (created at, updated at, etc.) from entering the array
+      if (!!entryMap.fields[field]) {
```

Comment:
I like this better than the other way we were doing it, where were declaring which ones to skip
`  const fieldsToSkip = ["updated_at", "updated_by_user_id"];`

butI like your way, because we dont have to declare which ones to skip, and it just skips anything that isnt defined. ✨